### PR TITLE
MDR40-004 Content(Front Page) -> Services

### PR DESCRIPTION
**Pull request by BryanPalad**

### CODE REVIEW

File: src/pages/Homepage/Services/Services.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
```
Comment (JMSevilla) [inline src/pages/Homepage/Services/Services.js:1]: Kindly re-construct your file structuring, As you can see we do have views folder so you can create your UI structure here. Also as you can see under signup folder inside views folder i have to files 

signup_fields.js
signup.jsx

Kindly retain this way of coding starting this sprint.

File: src/constants/Services/ServicesData.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import web from '../../assets/images/services/web_development.jpg';
+import mobile from '../../assets/images/services/mobile_app.jpg';
+import desktop from '../../assets/images/services/desktop_app.jpg';
+
+
+export const ServicesData = [ 
+    {
+        img: web,
+        title: 'Web App Development',
```
Comment (JMSevilla) [inline src/constants/Services/ServicesData.js:9]: Remove your constants folder and transfer your Service and Navbar Data to our utils helper.js

File: src/pages/Homepage/Services/Services.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import Box from '@mui/material/Box'
+import ServiceOutlineText from '../../../assets/images/outline-text/service.svg'
+import { ServicesData } from '../../../constants/Services/ServicesData'
+
+const Services = () => {
+
+  return (
+    <Box style={{display: 'flex', flexDirection: 'column', justifyContent: 'center', gap: '4.5rem'}}>
+        <Box style={{display: 'flex', flexDirection: 'column', alignItems: 'center', gap: '.5rem'}}>
+        <img src={ServiceOutlineText} alt='service outline'/>
```
Comment (JMSevilla) [inline src/pages/Homepage/Services/Services.js:15]: Transfer this whole code to your service_field.js then all our UI must fall in JSX

File: src/views/Home.jsx [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 import {Container} from '@mui/material'
-
+import Services from '../pages/Homepage/Services/Services'
 const Home = () => {
     return (
         <>
             <ApplicationBar title={'Ecommerce'} />
-            <Container style={{marginTop: '100px'}}>
-                test
+            <Container style={{marginTop: '150px'}}>
```
Comment (JMSevilla) [inline src/views/Home.jsx:9]: Use our system container

**Review by JMSevilla — CHANGES_REQUESTED** [review]
Hi bryan,

Kindly re-check your PR and check my reviews.

Thanks

File: src/constants/Services/ServicesData.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import web from '../../assets/images/services/web_development.jpg';
+import mobile from '../../assets/images/services/mobile_app.jpg';
+import desktop from '../../assets/images/services/desktop_app.jpg';
+
+
+export const ServicesData = [
```
[the remark this answers is not in the document]
Comment (BryanPalad) [reply src/constants/Services/ServicesData.js:6]: Noted and Resolved

File: src/pages/Homepage/Services/Services.js (hunk 3 above)
Comment (BryanPalad) [reply src/pages/Homepage/Services/Services.js:15]: Noted and Resolved.

File: src/views/Home.jsx (hunk 4 above)
Comment (BryanPalad) [reply src/views/Home.jsx:9]: Noted and Resolved.

File: src/components/Appbar/Appbar.js [hunk 6]
```diff
@@ -11,14 +11,52 @@ import { appRouter } from "../../routes/router";
...
+            buttonName={"Sign up"}
+            handleClick={() => navigateSignup()}
+            size={"small"}
+            color={"inherit"}
+          />
+        </Toolbar>
+      </AppBar>
+      </>):(<>
+      
+        <AppBar
```
[the remark this answers is not in the document]
Comment (BryanPalad) [reply src/components/Appbar/Appbar.js:59]: Noted and I assume it was resolved hehe.

**Review by JMSevilla — APPROVED** [review]

**Pull request merged**